Recently edited (#4)
apitoolkit/apitoolkit-landing: content/docs/Quickstarts/Golang.md

---
title: Golang 
date: 2022-09-09

> publishdate: 2022-09-09

Reviewer: I think you should not update publishdata for already existing pages

Author: okay

weight: 20
menu:
  main:
    weight: 20
---

> To integrate golang web services with API Toolkit, an SDK called the golang client for API Toolkit is utilized. It keeps track of incoming traffic, aggregates the requests, and then delivers them to the apitoolkit servers.We'll concentrate on providing a step-by-step instruction for integrating an API toolkit into our Golang web service in this tutorial. 

Reviewer: ```suggestion
To instrument golang web services with API Toolkit, we use the APIToolkit golang SDK. It keeps track of incoming traffic, aggregates the requests, and then delivers them to the apitoolkit servers.We'll concentrate on providing step-by-step instructions for integrating APIToolkit into a Golang web service in this tutorial. 
```

  2. Users to explore their API live, via the api log explorer.

## How to Integrate with Gin router:

> 1. First things first: Before integrating, always ensure that your codes are functioning properly.

Reviewer: I think this line is unnecessary

Author: Okay, I will take it off

> 2. Sign up / Sign in to the API dashboard

Reviewer: Shouldnt you have a page or proper walkthrough with screenshots about the process

Author: I had screenshots here, but deleting,  installing and all of that process made me lose it. so I just wanted to be sure I am on the right path.

> 3. Create a project, generate your API key, and include a brief description of your work. And to prevent losing your key after it has been generated, remember to make a copy of it. 

Reviewer: You should  be showing detailed walkthrough.

}
```
3. Deploy your application and enjoy exploring and managing your API via the APIToolkit dashboards and API log explorer

> Now this is what your dashboard should look like after a successful integration

Reviewer: Like what? There's no screenshot

Author: There was a screenshot here, that's why you have this here.
